src/Bijectors.jl

```diff
@@ -35,6 +35,9 @@ using MappedArrays
 using Base.Iterators: drop
 using LinearAlgebra: AbstractTriangular
 
+import ChangesOfVariables: with_logabsdet_jacobian
+import InverseFunctions: inverse
```

Comment:
Would it be possibly to not import these functions but qualify them when they are implemented? E.g., we could use here
```suggestion
using ChangesOfVariables: ChangesOfVariables, with_logabsdet_jacobian
using InverseFunctions: InverseFunctions, inverse
```
Then one could still call `with_logabsdet_jacobian` and `inverse` without qualifications but has to use `InverseFunctions.inverse` when implementing them.